crates/iota-core/src/epoch/randomness.rs

```diff
@@ -697,12 +697,11 @@ impl RandomnessManager {
         output: &mut ConsensusCommitOutput,
     ) -> IotaResult<Option<RandomnessRound>> {
         let epoch_store = self.epoch_store()?;
-        let tables = epoch_store.tables()?;
 
-        let last_round_timestamp = tables
-            .randomness_last_round_timestamp
-            .get(&SINGLETON_KEY)
-            .expect("typed_store should not fail");
+        let last_round_timestamp = epoch_store
+            .get_randomness_last_round_timestamp()
+            .expect("read should not fail");
```

Comment:
```suggestion
        let last_round_timestamp = epoch_store.get_randomness_last_round_timestamp()?;
```
Maybe error propagation is fine here?